lib/bandit/headers.ex

```diff
@@ -2,6 +2,8 @@ defmodule Bandit.Headers do
   @moduledoc false
   # Conveniences for dealing with headers
 
+  defguardp is_port_number(port) when Bitwise.band(port, 0xFFFF) === port
+
```

Comment:
Nit, but if this is private oughtn't it be at the bottom of the file?

Reply:
guards have to be defined above where you use them

Reply:
^